extjsdk/src/test/java/io/vantiq/extjsdk/TestExtensionWebSocketClient.java

```diff
@@ -511,9 +505,21 @@ public boolean receivedMessage() {
         }
 
         @Override
-        public void sendPing(Buffer payload) throws IOException {}
+        public boolean close(int code, String reason) {
+            client.getListener().onClosed(null, code,reason);
```

Comment:
Seems like instead of `null` for the WebSocket, we can pass in `this` can't we?  Seems like that would be cleaner, since it is a `WebSocket`.